### PR TITLE
Issue 76: additions to Asciidoc chapter

### PR DESCRIPTION
**Pull request by inoxx03**
@ncbaratta I've put all sentences on a new line to improve source readability, otherwise, I've made no changes to your content:)

I've added a section with some tips on using attributes to single source book local and external links.

I also added link sto useful resources related to previewing AsciiDoc content locally.

Please let me know your thoughts.

### CODE REVIEW

File: topics/asciidoc.adoc [hunk 1]
```diff
@@ -1,15 +1,21 @@
-[[asciidoc]]
+[asciidoc]
```
Comment (ncbaratta) [inline topics/asciidoc.adoc]: Does this syntax work? I thought it had to be:

[[anchor]]
[#anchor]
or
[id="anchor"]

File: topics/asciidoc.adoc [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+AsciiDoc files can be translated to many formats including HTML, PDF, EPUB, and man page.
 
 The Red Hat documentation team uses http://asciidoctor.org/[Asciidoctor^] as part of the publishing toolchain.
 
-[[ccg-asciidoc-attributes]]
+[ccg-asciidoc-attributes]
 === AsciiDoc Attributes
 
-AsciiDoc attributes are variables you can use in common files to avoid hard-coding brand-specific information. Using attributes allows you to more easily share content between multiple brands. To use attributes, first define them in the relevant _brand_ AsciiDoc attribute file in the `common` directory. Then, include the _brand_ AsciiDoc attribute file in the `master.adoc` file for the relevant document. Finally, replace brand-specific terms with the variable names that you have defined, and use `ifdef` statements to indicate mark blocks of brand-specific content.
+AsciiDoc attributes are variables you can use in common files to avoid hard-coding brand-specific information.
```
Comment (ncbaratta) [inline topics/asciidoc.adoc]: Should we make this a numbered list/procedure?

File: topics/asciidoc.adoc [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
+Using attributes allows you to more easily share content between multiple brands.
```
Comment (ncbaratta) [inline topics/asciidoc.adoc]: Should we make this a numbered list/procedure?

File: topics/asciidoc.adoc [hunk 4]
```diff
@@ -57,11 +64,61 @@ include::common/attributes/community.adoc
...
+--
+
+Use the variable in a link macro.
+The variable will automatically resolve to the URL defined in the attribute when you build your documents.
+
+[source,asciidoc,options="nowrap",subs="attributes+"]
+--
+link:{link-community-docs}#introduction[Community Documentation Introduction^]
+--
+TIP: Use the caret `^` character to enforce the `target=_blank` attribute for the link anchor.
```
Comment (ncbaratta) [inline topics/asciidoc.adoc]: Actually this isn't true, it's a combination of including the :linkattrs: value and using the caret (http://asciidoctor.org/docs/asciidoc-writers-guide/#links-and-images)

Comment (inoxx03) [reply topics/asciidoc.adoc]: @ncbaratta Thank you, You're right, without `linkattrs`, it wouldn'y work. Let me add that information.

File: topics/asciidoc.adoc (hunk 1 above)
Comment (inoxx03) [reply topics/asciidoc.adoc]: @ncbaratta I tested and while it works with AsciiDoctor, it breaks the TOC when built using CCUTIL.
`[#anchor]` works consistently with AsciiDoctor and CCUTIL.
I'll fix the anchor formatting.

File: topics/asciidoc.adoc (hunk 3 above)
Comment (inoxx03) [reply topics/asciidoc.adoc]: I agree, let's do it.

**Comment by inoxx03** [comment]
@ncbaratta Thank you for the review, Ive incorporated your suggestions and added some examples, typo corrections and formatting fixes.
Please let me know what you think.

**Pull request merged**